src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java

```diff
@@ -0,0 +1,151 @@
+package org.geoserver.geofence.rest;
```

Comment:
Hi Niels

I  did it like the other REST stuff in geoserver, I have no idea about
using Spring for the REST API.

On Sat, May 16, 2015 at 9:51 AM, NielsCharlier <notifications@github.com>
wrote:

> In
> src/community/geofence-server/src/main/java/org/geoserver/geofence/rest/UsersRestController.java
> <https://github.com/geoserver/geoserver/pull/1044#discussion_r30459367>:
>
> > @@ -0,0 +1,151 @@
> > +package org.geoserver.geofence.rest;
>
> perhaps we should make a new module for spring rest, like restconfig-ng as
> we move things gradually. @jodygarnett <https://github.com/jodygarnett>
> please advise
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoserver/geoserver/pull/1044/files#r30459367>.
>



-- 
DI Christian Mueller MSc (GIS), MSc (IT-Security)
OSS Open Source Solutions GmbH